src/phospho.ts

```diff
@@ -35,24 +94,40 @@ class Phospho {
     }
     if (tick) this.tick = tick;
     if (baseUrl) this.baseUrl = baseUrl;
+    if (path_to_hash) {
+      this.version_id = hashCode(path_to_hash);
+    } else {
+      path_to_hash = process.cwd();
+      this.version_id = hashCode(path_to_hash);
+    }
```

Comment:
Are you sure about this fallback ? 
There is also a default version_id server-side that is happening when version_id is None. 
I think this default version_id based on a hash of the cwd should be applied _if and only if_ pathToHash is null, and not if pathToHash is undefined. 
What do you think ?